Library/Homebrew/global.rb

```diff
@@ -26,9 +25,6 @@
 require "active_support/core_ext/string/exclude"
 require "active_support/core_ext/string/indent"
 
-I18n.backend.available_locales # Initialize locales so they can be overwritten.
-I18n.backend.store_translations :en, support: { array: { last_word_connector: " and " } }
```

Comment:
I missed this detail in opening https://github.com/Homebrew/brew/pull/14847/files#diff-22cac98de67fa59dbc9c0011c7d85c4b204f77eecb7c2e2a7f5a5a0ac16ec823 which is why it has some minor changes around the oxford comma. (I can change the default back, but even as someone who dislikes the oxford comma, i don't think it's worth the trouble.)

Reply:
@dduugg It'd be nice to change the default back into the relevant array/pluralize case in this PR, thanks!

Reply:
done!